config/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+
+// @see https://docs.newrelic.com/docs/agents/nodejs-agent/installation-configuration
+require('newrelic');
```

Comment:
Actually, to `server.js`, as it is the main entrypoint: https://github.com/DoSomething/gambit/blob/87c3199e2de01929de55a630628bedaaff39fd1c/Procfile#L1.